pynucastro/rates/tests/test_rates.py

```diff
@@ -244,6 +260,21 @@ def test_identical_particle_factor(self):
 
         assert self.rate8.prefactor == 1.0
 
+    def test_stoichiometry(self, reaclib_library):
+        assert repr(self.rate4) == "C12 + He4 ⟶ O16 + 𝛾"
+
+        # create a separate version since rates are mutable
+        c12ag = reaclib_library.get_rate_by_name("c12(a,g)o16")
+        c12ag.stoichiometry = {Nucleus("he4"): 1.5,
+                               Nucleus("c12"): 1,
+                               Nucleus("o16"): 1}
+        c12ag._set_print_representation()  # pylint: disable=protected-access
```

Comment:
I think we should automate via setter? do `_set_print_representation()` whenever stoichiometry is set.

Reply:
I don't understand this comment

Reply:
I wasn't really concerned about this since it is just in a test